common.c

```diff
@@ -1101,6 +1102,27 @@ trace_mode(mode_t mode, char *p)
 	*p = '\0';
 }
 
+/* printf backtrace callback */
+void trace_printf_backtrace(void)
+{
+	void *callstack[128];
+	int old_trace_state;
+
+	old_trace_state = trace_disable();
+
+	int i, frames = backtrace(callstack, 128);
+	char **strs = backtrace_symbols(callstack, frames);
```

Comment:
Here I see one challenge: `backtrace_symbols` is using inside malloc for example at [gnu libc](https://github.com/lattera/glibc/blob/master/debug/backtracesyms.c#L71).

And it's should create an infinity loop of printing stacktrace.